src/IsoCodes/Insee.php

```diff
@@ -110,6 +110,6 @@ public static function validate($numero)
             return false;
         }
 
-        return true;
+        return $return;
```

Comment:
:-1: for several reasons:

* It's currently a BC break.
* IMHO return mixed type when we are able to avoid it is a bad practice, especially since the possibility of strict typing.
We may pass an array as reference instead to fill the data. And this array must be optional.
The another solution would be to have a separate method like `getInfo(string $insee)` for example.